docs/index.html

```diff
@@ -2,9 +2,7 @@
 layout: v2
 ---
 
-<div class="announcement">Nov 21, 2016 — <a href="http://leafletjs.com/2016/11/21/leaflet-1.0.2.html">Leaflet 1.0.2</a>, a bugfix release, is out.</div>
-
-<div class="announcement">Sep 27, 2016 — <a href="http://leafletjs.com/2016/09/27/leaflet-1.0-final.html">Leaflet 1.0</a>, the fastest, most stable and polished Leaflet release ever, is finally out!</div>
+<div class="announcement">Jan 23, 2017 — <a href="http://leafletjs.com/2016/11/21/leaflet-1.0.3.html">Leaflet 1.0.3</a>, a bugfix release, is out.</div>
```

Comment:
Damn, missed that the link was broken here. Fixed in https://github.com/Leaflet/Leaflet/commit/174e693912e3eec0434af4fd9bdb96f1351bac8b @IvanSanchez